packages/mobile/src/components/core/Text.tsx

```diff
@@ -0,0 +1,25 @@
+import { Text as TextBase, TextProps as TextBaseProps } from 'react-native'
```

Comment:
```suggestion
import { Text as RNText, TextProps as RNTextProps } from 'react-native'
```

Just to stay consistent with other places